scout/build/disease.py

```diff
@@ -42,7 +42,6 @@ def build_disease_term(disease_info: dict, alias_genes: dict = {}) -> dict:
 
     disease_obj["genes"] = list(hgnc_ids)
 
-    LOG.warning(disease_obj)
```

Comment:
good stuff though.. 😸 I assume we will have a different PR for it?

Reply:
yes!